detect/buildpacks.go

```diff
@@ -116,13 +117,19 @@ var _ = DetectDescribe("Buildpacks", func() {
 	})
 
 	Describe("dotnet-core", func() {
-		It("makes the app reachable via its bound route", func() {
-			Expect(cf.Cf("push", appName, "-m", DEFAULT_MEMORY_LIMIT, "-p", assets.NewAssets().DotnetCore).Wait(Config.DetectTimeoutDuration())).To(Exit(0))
-
-			Eventually(func() string {
-				return helpers.CurlAppRoot(Config, appName)
-			}).Should(ContainSubstring("Hello from dotnet-core"))
-		})
+		for _, stack := range Config.GetStacks() {
+			It(fmt.Sprintf("[%s] makes the app reachable via its bound route", stack), func() {
+				assetPath, ok := assets.NewAssets().DotnetCore[stack]
+				if !ok {
```

Comment:
Please change this to an expectation, i.e. `Expect(ok).To(BeTrue())`. You can still add your current failure message to that [as an annotation](https://onsi.github.io/gomega/#annotating-assertions).